lib/plottr_components/src/components/PlottrFloater.js

```diff
@@ -35,7 +36,17 @@ const PlottrFloaterConnector = (connector) => {
             arrowColor={darkMode ? '#555' : 'white'}
             arrowSize={10}
             arrowStyle={{}}
-            className="popover-arrow-container"
+            className={cx('popover-arrow-container', {
+              resize:
```

Comment:
So does this check whether the component is anchored to the middle of the window and occupies the other half of the window?